src/classes/GE_GiftEntryController.cls

```diff
@@ -177,6 +177,52 @@ public with sharing class GE_GiftEntryController {
         }
     }
 
+    @AuraEnabled
```

Comment:
@kenlewis-sfdo what do you think about having a TODO comment here that mentions that this is a temporary canned response that will be replaced with the real API callout in time? I know we attempt to avoid comments, but just from the review perspective it's nice to see that intention represented here 

(I assume that's the intention here, otherwise ... I have some issue with this!)

Reply:
@ptewson-sfdo yeah definitely temporary

Reply:
Ideally, while in development, this method would new up an instance of `CaptureGroupStub` and call `add` on that object, which would return these static responses.

Then, when ready to make real callouts, this method switches to using the real `CaptureGroup`.